core/spec.md

```diff
@@ -436,6 +445,114 @@ attributes. However they MUST adhere to the following rules:
   potential conflicts with future Registry specification attributes. For
   example, use of a model (or domain) specific prefix could be used.
 
+#### `xid` - Referencing xRegistry Entities
+
+An `xid` is a URI according to
+[RFC 3986](https://datatracker.ietf.org/doc/html/rfc3986) that
+uniquely identifies an entity within an xRegistry. Its purpose is to
+reference the entity and express a relation to it, even if it is not stored in
+the local registry. The `xid` can also be used in other places where it
+is helpful to use a URI, e.g. events or knowledge graphs.
+Each entity has exactly one `xid`. Therefore, comparing two `xid`
+values is possible to determine, if they identify the same or different
+entities. To achieve this, an xRegistry-specific URI scheme `xreg` is defined:
+
+`xid` = `"xreg://" authority "/" entity-path "/" entity-id`
```

Comment:
Looking at some examples:
```
xreg://example.com/schemagroups/g1/schemas/s1/versions/v1
xreg://example.com/schemagroups/g1/schemas/s1
xreg://example.com/schemagroups/g1
xreg:///schemagroups/g1
```
Is there a way to reference the registry itself? e.g. `xreg:///regID` or `xreg:////regID` (a lot of /'s) ? Do we think that might be needed in the future? Seems like it.

Does `xreg://example.com/` mean anything? I guess it could be a registry, but w/o an ID you can only have one per authority which doesn't feel right. Would we need to include the registry ID only when there's an authority?

we may want to say that the format used here is BNF (or eBNF).